spec/commands/add_spec.rb

```diff
@@ -28,6 +30,18 @@
       expect(bundled_app("Gemfile").read).to match(/gem 'baz', '~> 1.2'/)
       expect(the_bundle).to include_gems "baz 1.2.3"
     end
+
+    it "version requirement becomes ~> major.minor.patch.pre when resolved version is < 1.0" do
```

Comment:
@segiddins Is this the correct expectation when a pre-release version is resolved?